invenio_records_marc21/__init__.py

```diff
@@ -8,7 +8,107 @@
 # modify it under the terms of the MIT License; see LICENSE file for more
 # details.
 
-"""Marc21 datamodel."""
+"""Invenio-Records-Marc21 datamodel.
+
+This guide will show you how to get started with
+Invenio-Records-Marc21. It assumes that you already have knowledge of
+Flask applications and Invenio modules.
+
+It will then explain key topics and concepts of this module.
+
+Getting started
+---------------
+
+You will learn how to create a new Marc21 records, upload a File and accosiate with the record
+using the programmatic APIs of Invenio-Records-Marc21.
+
+
+This is the initial configuration needed to have things running:
+
+:py:data:`invenio_records_marc21.config`
+
+
+Service operations
+------------------
+
+Creation
+~~~~~~~~
+
+
+Let's **create** a very simple record:
+
+>>> from flask import Flask, g
+>>> app = Flask(__name__)
+
+>>> from invenio_records_marc21 import InvenioRecordsMARC21
+>>> ext = InvenioRecordsMARC21(app)
+
+>>> from invenio_records_marc21.proxies import current_records_marc21
+>>> service = current_records_marc21.record_resource
+>>> draft = service.create(identity=g.identity,data={"metadata": {"title": "The title of the record"}})
```

Comment:
please add realistic minimal marc21 metadata:

data={"metadata": {"title_statement": {"title": "The title of the record"}}}